src/shared/components/Checkbox/Checkbox.styles.tsx

```diff
@@ -108,4 +107,13 @@ export const Checkmark = styled.div`
   width: 18px;
   height: 18px;
   border-radius: 1px;
+  position: relative;
+  svg {
+    position: absolute;
+  }
```

Comment:
Why do we need this?

Reply:
![check](https://user-images.githubusercontent.com/51168865/114160033-be925480-9926-11eb-9cc8-5e86c0b12745.gif)
I had an odd layout shift before, but I guess it's fixed now.